sources/web/datalab/templates/edit.html

```diff
@@ -88,7 +88,9 @@
     window.datalab = {};
 
     $("#appBar").load("<%baseUrl%>static/appbar.html", function() {
-      requirejs(['edit/js/main.min']);
+        requirejs(['websocket'], (websocket) => {
```

Comment:
Why don't we add `'edit/js/main.min'` to the list of dependencies of the `websocket` module? Is there a scenario where that doesn't work?

Reply:
I see. LGTM then.